src/main/java/nl/rijksoverheid/mev/gezagsmodule/domain/Persoonslijst.java

```diff
@@ -30,236 +27,116 @@
-            return (List<GeschiedenisOuder1>) (Object) listValues.get(Categorie.GESCHIEDENIS_OUDER_1);
-        } else {
-            return new ArrayList<>();
-        }
-    }
-
-    public List<GeschiedenisOuder2> getGeschiedenisOuder2() {
-        if (listValues.containsKey(Categorie.GESCHIEDENIS_OUDER_2)) {
-            return (List<GeschiedenisOuder2>) (Object) listValues.get(Categorie.GESCHIEDENIS_OUDER_2);
-        } else {
-            return new ArrayList<>();
-        }
+        inschrijving = new Inschrijving(lo3PlRecord);
     }
 
-    public static final int MEERDERJARIGE_LEEFTIJD = 180000;
-    public static final int AKTE_NUMMER_LENGTE = 3;
-    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("yyyyMMdd");
-    /*
-        In het BRP wordt een puntouder vastgelegd in de geslachtsnaam met een "."
-     */
-    public static final String GEEN_OUDERS = "Geen_ouders";
-    public static final String EEN_OUDER = "Een_ouder";
-    public static final String TWEE_OUDERS = "Twee_ouders";
-    public static final String GESLACHTNAAM_AANDUIDING_PUNTOUDER = ".";
-
-    /*
-        Mogelijke antwoorden op de vraag: Hoeveel Juridische Ouders
-     */
-    public static final String PUNTOUDERS = "1_of_2_puntouders";
-    /*
-        Akte aanduidingen die gebruikt worden om adoptie of erkenning vast te stellen
-        Vaderschap is toegevoegd om in situaties van 1 ouderlijk gezag de gezaghebbende ouder te kunnen bepalen
-     */
-    public static final char TABEL_39_AKTEAANDUIDING_ERKENNING_BIJ_DE_GEBOORTE_AANGIFTE = 'B';
-    public static final char TABEL_39_AKTEAANDUIDING_ONTKENNING_OUDERSCHAP = 'E'; // Rechterlijke uitspraak
-    public static final char TABEL_39_AKTEAANDUIDING_ERKENNING_NA_DE_GEBOORTEAANGIFTE = 'C';
-    public static final char TABEL_39_AKTEAANDUIDING_NOTARIELE_AKTE_VAN_ERKENNING = 'J';
-    public static final char TABEL_39_AKTEAANDUIDING_ADOPTIE = 'Q';
-    public static final char TABEL_39_AKTEAANDUIDING_GEBOORTE = 'A';
-    public static final char TABEL_39_AKTEAANDUIDING_GERECHTELIJKE_VASTSTELLING_OUDERSCHAP = 'V';
-    public static final String PUNTOUDER_INDICATIE = ".";
-    static final List<String> INDICATIE_GEZAG_CODES = Arrays.asList("1", "2", "12", "1D", "2D", "D");
-    static final String[] HUWELIJK_OF_PARTNERSCHAP_REDEN_ONTBINDING = new String[]{"O", "R"};
-
-    @JsonIgnore
-    @Getter
-    private HopRelaties hopRelaties = null;
-
-    @Getter
-    private String receivedId;
-
-    public List<String> getBurgerservicenummersVanKinderen() {
-        if (getKinderen() == null) {
-            return Collections.emptyList();
-        }
-
-        return getKinderen().stream()
-            .map(Kind::getBsn)
-            .filter(Objects::nonNull)
-            .toList();
+    public Set<String> getUsedVeldenInOnderzoek() {
+        return Stream.of(
+                persoon != null ? persoon.getVeldenInOnderzoek().stream() : null,
```

Comment:
Hierdoor komen `null`s in de Stream/Set; liever voorkomen.

Besproken tijdens meeting.